src/app.ts

```diff
@@ -83,7 +83,10 @@ async function app() {
   const dependencyDefs = loadExternalDependencies(defs, config);
 
   // Load custom resources specified in ig-data folder
-  loadCustomResources(input, defs);
+  loadCustomResources(path.join(input, 'ig-data', 'input'), defs);
+  if (isIgPubContext) {
+    loadCustomResources(path.join(input, '..', 'input'), defs);
+  }
```

Comment:
I feel like this could make for some really weird situations.  If you have definitions in `ig-data/input` then once you run SUSHI one time, those definitions will be in `../input`.  And then the next time you run SUSHI, it will load the definitions from `ig-data/input` and then load the same definitions from `../input`.

As a result, I could see the potential for two bad things (if I understand this correctly):

1. If you edit a file in `ig-data/input`, but the same file already exists in `../input` (due to a recent SUSHI run) -- then won't the old unedited file load over the new edited file (since `../input` gets loaded after `ig-data/input`)?

2. If you delete a file in `ig-data/input` but it still exists in `../input` (due to a recent SUSHI run), then SUSHI will load the old one from `../input` and then add it to the IG JSON `resources` list -- effectively making it as if it was not deleted at all.

To me, it seems like if `ig-data` exists, then we best just load stuff from there (and there only).  We should only load directly from `../input` when the author has intentionally chosen not to use `ig-data`.  Does that make sense?

Reply:
Yeah that does make sense, I'll make that change!

Reply:
Changed in https://github.com/FHIR/sushi/pull/581/commits/5b2a5223cd356a9aae1e0a1001aaa6ed2207efc4.